en/middleware/micrortps.md

```diff
@@ -1,44 +1,47 @@
-# RTPS/ROS2 Interface: PX4-FastRTPS Bridge
+# RTPS/DDS Interface: PX4-Fast RTPS(DDS) Bridge
 
-The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast RTPS* applications (including those built using the ROS2/ROS frameworks).
+The *PX4-Fast RTPS(DDS) Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast DDS* applications.
```

Comment:
FYI, subedited this to make it more readable. I slightly de-emphasise ROS 2 here, pointing to the other doc for that. Please check.

Reply:
Yep looks goof! Thanks @hamishwillee